src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -77,9 +77,9 @@ public Collection<string> ArgumentList
 
         public bool CreateNoWindow { get; set; }
 
-        public StringDictionary EnvironmentVariables => new StringDictionaryWrapper(Environment as DictionaryWrapper);
+        public StringDictionary EnvironmentVariables => new StringDictionaryWrapper((Environment as DictionaryWrapper)!);
```

Comment:
Maybe use explicit casting?

Reply:
Explicit casting will change IL. Is it acceptable for such PR type?

Reply:
We are doing such changes, by my understdanding changes which wouldn't increase IL size